runtime/crab-parachain/src/lib.rs

```diff
@@ -349,6 +349,15 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 
+	impl pallet_fee_market_rpc_runtime_api::FeeMarketApi<Block, Balance> for Runtime {
+		fn market_fee() -> Option<pallet_fee_market_rpc_runtime_api::Fee<Balance>> {
+			Default::default()
```

Comment:
Curious why this API implementation is empty.

Reply:
No s2s bridge for Crab/Darwinia.

Reply:
I am not aware of the reason~, can you explain more about this? The bridge-messages pallets added in this pr will be used in Crab<> Crab parachain. What's the relationship with Darwinia Chain here?

Reply:
For Pangolin<>Pangolin Parachain. Not for Crab/Darwinia.

Reply:
suggest adding some comments like `TOO: xxx`